tests/fsharpqa/Source/CodeGen/EmittedIL/Misc/LetIfThenElse01.il.netfx4.bsl

```diff
@@ -65,121 +65,113 @@
              [6] int32 y2,
              [7] valuetype [mscorlib]System.DateTime V_7)
     .language '{AB4F38C9-B6E6-43BA-BE3B-58080B2CCCE3}', '{994B45C4-E6E9-11D2-903F-00C04FA302A1}', '{5A869D0B-6611-11D3-BD2A-0000F80849BD}'
-    .line 6,6 : 7,9 
+    .line 6,6 : 12,51 'C:\\GitHub\\dsyme\\visualfsharp\\tests\\fsharpqa\\Source\\CodeGen\\EmittedIL\\Misc\\LetIfThenElse01.fs'
     IL_0000:  nop
-    IL_0001:  nop
```

Comment:
In this example, a ``nop`` instruction has been removed, which causes a larger diff